p2p/src/main/kotlin/maru/p2p/P2PNetworkImpl.kt

```diff
@@ -31,8 +31,8 @@ import tech.pegasys.teku.networking.p2p.libp2p.PeerAlreadyConnectedException
 import tech.pegasys.teku.networking.p2p.network.PeerAddress
 import tech.pegasys.teku.networking.p2p.peer.DisconnectReason
 import tech.pegasys.teku.networking.p2p.peer.NodeId
-import tech.pegasys.teku.networking.p2p.peer.Peer
 import tech.pegasys.teku.networking.p2p.rpc.RpcStreamController
+import tech.pegasys.teku.networking.p2p.peer.Peer as TekuPeer
 
 class P2PNetworkImpl(
```

Comment:
This probably needs some coordination with Stefan's [PR](https://github.com/Consensys/maru/pull/126)